jupyter-notebooks/scipy-2022-workshop/5_plotting_a_histogram.ipynb

```diff
@@ -0,0 +1,185 @@
+      "text/plain": [
+       "<Figure size 1440x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Define a new figure\n",
+    "fig2 = plt.figure(figsize=(20,10))\n",
+    "\n",
+    "# Give this new figure a subplot, which will contain the histogram itself\n",
+    "ax = fig2.add_subplot(111)\n",
+    "\n",
+    "# Add a title & (x,y) labels to the plot\n",
+    "plt.title(\"Histogram Example\", fontsize=18, fontweight='bold')\n",
+    "plt.xlabel(\"pixel values\", fontsize=14)\n",
+    "plt.ylabel(\"Number of pixels\", fontsize=14)\n",
+    "\n",
+    "# let's plot non-null blue band values as an example\n",
+    "# For the x-axis, we want to count every pixel that has a value\n",
+    "x = blue[np.not_equal(blue, satdat.nodata)]\n",
+    "\n",
+    "# Define the number of bins to divide the data into\n",
+    "bins = 50\n",
+    "\n",
+    "# Define a color for the histogram\n",
+    "# You can use https://matplotlib.org/2.0.0/examples/color/named_colors.html as a reference\n",
+    "color = 'lightgreen'\n",
```

Comment:
Since you're looking at blue-band data, why not set the colour to some shade of blue? Perhaps choose your favourite shade from https://matplotlib.org/stable/gallery/color/named_colors.html - my go-to is `color = royalblue`, but chef's choice!